spec/GraphQLOverHTTP.md

```diff
@@ -745,3 +745,37 @@ payload is `application/json` then the client MUST NOT rely on the body to be a
 well-formed _GraphQL response_ since the source of the response may not be the
 server but instead some intermediary such as API gateways, proxies, firewalls,
 etc.
+
+# Additional Notes
```

Comment:
[In the main spec](https://spec.graphql.org/draft/#sec-Non-Normative-Portions), all text is normative except otherwise sepcified. Might be worth being explicit?

```suggestion
# Non-Normative Notes
```